pyproject.toml

```diff
@@ -7,7 +7,7 @@ build-backend = "setuptools.build_meta"
 
 [metadata]
 name = "oidcop"
-version = "2.1.0"
+version = "2.3.0"
```

Comment:
mmhhh... it's already in src/.__init__.py

Reply:
Don't know which has precedence.

Reply:
we need a strategy here to prevent to update the same value in two different place/files

https://pypi.org/project/poetry-dynamic-versioning/

That would be more like an issue to be implemented in the near future

Reply:
Yes ! We're using poetry för CryptoJWT and it's frustrating moving between that framework and the one we
use for all the others. I think we should eventually move to poetry for all libraries.